lib/syntax/node/CDC.js

```diff
@@ -1,7 +1,7 @@
 import { CDC } from '../../tokenizer/index.js';
 
 export const name = 'CDC';
-export const structure = [];
+export const structure = {};
```

Comment:
CDC and CDO were the only nodes that had empty arrays for `structure`, so I normalized to objects.